src/System.Management.Automation/engine/NativeCommandProcessor.cs

```diff
@@ -1714,7 +1768,18 @@ internal class ProcessOutputHandler
         private bool _isXmlCliError;
         private readonly string _processFileName;
 
+        private readonly AsyncByteStreamDrainer _stdOutDrainer;
+
         public ProcessOutputHandler(Process process, BlockingCollection<ProcessOutputObject> queue)
+            : this(process, queue, null, out _)
+        {
+        }
+
+        public ProcessOutputHandler(
+            Process process,
+            BlockingCollection<ProcessOutputObject> queue,
+            BytePipe stdOutDestination,
+            out AsyncByteStreamDrainer stdOutDrainer)
```

Comment:
I'm trying not to *too* drastically alter the classes here but `ProcessOutputHandler` is a bit non-typical already. We were just creating an instance of it and not saving it anywhere so the constructor sort of functioned like a static method. I need to take a closer look at if it's worth rethinking this class in general as this out param doesn't help things.